spyder/utils/icon_manager.py

```diff
@@ -10,12 +10,16 @@
 # Third party imports
 from qtpy.QtGui import QIcon
 from qtpy.QtWidgets import QStyle, QWidget
+from qtconsole.styles import dark_color
 
 # Local imports
 from spyder.config.base import get_image_path
 from spyder.config.main import CONF
+from spyder.config.gui import get_color_scheme
 import qtawesome as qta
 
+# Main color for a dark theme
+MAIN_LIGHT_COLOR = 'white'
```

Comment:
I think the description I putted is missliding. I will delete it. In that way ´MAIN_LIGHT_COLOR´ is more general concept and maybe could be thought as a property to make the icons look clear or 'light' (in the sens that they contrast with a dark background).

Reply:
>  In that way ´MAIN_LIGHT_COLOR´ is more general concept and maybe could be thought as a property to make the icons look clear or 'light' (in the sens that they contrast with a dark background).

Okay, sure—that works.

Reply:
> 
> 
> Can't we use this variable to define a single icons dictionary?

It could be possible, then the validations in the icon function will be in the top when defining the variable which then should be called something like ´MAIN COLOR´.